lang/en/auth_saml2.php

```diff
@@ -108,6 +108,16 @@
 $string['manageidpsheading'] = 'Manage available IdPs';
 $string['mdlattr_help'] = 'Which Moodle user field should the IdP attribute be matched to?';
 $string['mdlattr'] = 'Mapping Moodle';
+$string['wantassertionssigned'] = 'Want Assertions Signed';
+$string['wantassertionssigned_help'] = '';
+$string['assertionsconsumerservices'] = 'Assertions Consumer Services';
+$string['assertionsconsumerservices_help'] = '';
+$string['entityid'] = 'Entity ID';
+$string['entityid_help'] = '';
```

Comment:
gonna sound like a broken record - this needs docs, none of these should be empty. This one I think its quite important to tell people that in 99.9% of cases they can leave it blank

Reply:
yep added the help texts, should this be documented anywhere else?